app/services/resolve_account_service.rb

```diff
@@ -118,10 +118,10 @@ def handle_activitypub
   def create_account
     Rails.logger.debug "Creating new remote account for #{@username}@#{@domain}"
 
-    @account = Account.new(username: @username, domain: @domain)
-    @account.suspended   = true if auto_suspend?
-    @account.silenced    = true if auto_silence?
-    @account.private_key = nil
+    @account              = Account.new(username: @username, domain: @domain)
```

Comment:
I don't agree with this whitespace change but it's also not a huge deal. this line is semantically different from the lines that follow, so they shouldn't be grouped together